_lib/wordpress_post_processor.py

```diff
@@ -30,46 +29,39 @@ def documents(name, url, **kwargs):
-    post['_id'] = post['slug']
-    # Set the proper keys for Newsroom
-    if post['type'] == 'cfpb_newsroom':
-        post['category'] = [cat['title'].replace('&amp;', '&')
-                            for cat in
-                            post['taxonomy_cfpb_newsroom_cat_taxonomy']]
-        post['author'] = [author['title'] for author in
-                          post['taxonomy_fj_author'] if 'Press Release' not in
-                          post['category']]
-    # Set the proper keys for Blog
-    elif post['type'] == 'post':
-        post['blog_category'] = [cat['title'].replace('&amp;', '&') for cat in
-                                 post['taxonomy_fj_category']]
-        post['category'] = ['Blog']
-        post['author'] = [author['title'] for author in
-                          post['taxonomy_fj_author']]
-    # Set the proper keys for Featured Topic
-    if post['type'] == 'featured_topic':
-        post['author'] = [post['author']['name']]
-        # convert featured_topic_data_x into a proper list
-        links = []
-        for x in xrange(0, 10):
-            custom_fields = post['custom_fields']
-            key = 'featured_topic_data_%s_link' % x
-            if key in custom_fields:
-                links.append(custom_fields[key])
-        post['links'] = links
-    else:
-        post['tags'] = [tag['title'] for tag in post['taxonomy_fj_tag']]
+    del post["comments"]
+    post["_id"] = post["slug"]
+    post["blog_category"] = [cat["title"].replace("&amp;", "&") for cat in
+                             post["taxonomy_fj_category"]]
+    post["category"] = ["Blog"]
+    post["author"] = [author["title"] for author in
+                      post["taxonomy_fj_author"]]
+    post["tags"] = [tag["title"] for tag in post["taxonomy_fj_tag"]]
+
+    names = ["og_title", "og_image", "og_desc", "twtr_text", "twtr_lang",
+             "twtr_rel", "twtr_hash", "utm_campaign", "utm_term",
+             "utm_content", "alt_title", "popular_posts",
+             "show_featured_image_in_post", "display_in_newsroom",
+             "related_links", "dsq_needs_sync", "dsq_thread_id"]
+    for name in names:
+        if name in post["custom_fields"]:
+            post[name] = post["custom_fields"][name]
+
+    if "related_hero" in post["custom_fields"]:
+        if "str" in str(type(post["custom_fields"]["related_hero"])):
+            post["related_hero"] = post["custom_fields"]["related_hero"]
+        else:
+            post["related_hero"] = post["custom_fields"]["related_hero"][0]
+
+    if "related_links" not in post:
+        related = []
+        for x in xrange(0, 5):
+            key = "related_links_%s" % x
+            if key in post["custom_fields"]:
+                related.append({"url": post["custom_fields"][key][0],
+                                "label": post["custom_fields"][key][1]})
+        post["related_links"] = related
+
+    del post["custom_fields"]
 
-    dt = dateutil.parser.parse(post['date'])
```

Comment:
Since we already have some temporary code to support the older version, what are your thoughts on including a temporary conditional to fix the date?  That way we'd be able to index properly in the mean time.